### PR TITLE
Fix book.id undefined bug

### PR DESCRIPTION
**Pull request by bt-s**

### CODE REVIEW

File: src/components/Books/SearchResults.js [hunk 1]
```diff
@@ -8,7 +8,7 @@ const SearchResults = props => (
     {props.books.map((book, ix) => (
       <div key={ix} className="book-item-container">
         <BookItem
-          bookId={book.id}
+          bookId={book.id || book.objectID}
```
Comment (Envl) [inline src/components/Books/SearchResults.js]: We dont have such a field in our db

Comment (Envl) [reply src/components/Books/SearchResults.js]: Don't use this. Maybe you wrote this in firebase Dashboard manually?
Although u can not see the id in firebase dashboard. But it is provided in the fetched data.

**Review by Envl — CHANGES_REQUESTED** [review]
Cancel this PR.

**Review by Envl — APPROVED** [review]

**Comment by Envl** [comment]
Did it in improve history branch

**Review by fadhilmch — APPROVED** [review]

**Pull request merged**